app/controllers/articles_controller.rb

```diff
@@ -19,6 +21,8 @@ def destroy
   end
 
   def update
+    params[:article][:categories].shift
```

Comment:
I still don't get what it does for you so I can't approve it.

Reply:
Read above answer

Reply:
I did.  That answer is NOT bringing clarity to the issue imo. I still don’t get it. Will move on though.